### ISSUE TITLE
[Potential Bug] ETA in the past?

### ISSUE THREAD

**Issue by jay-tux** [aisstream/issues#30]
After looking at a few vessels in the stream, it seems like the AIS ETA is always in the past (often by multiple days). Any idea how or why?

**Comment by aisstream** [comment]
This may be a bug on our end. We will look into this.

**Comment by jay-tux** [comment]
Update - not always, but quite often. Is this misreporting by the ships themselves?

**Comment by aisstream** [comment]
Yes, it is likely it is mis-reporting. We do not touch that value but I will need to verify our time recording isn't doing any incorrect conversions.
